elisp/mew-pop.el

```diff
@@ -39,7 +39,7 @@
 ;;;
 
 (defvar mew-pop-fsm
-  '(("greeting"      nil ("\\+OK" . "capa"))
+  '(("greeting"      t   ("\\+OK" . "capa"))
     ("capa"          t   ("\\+OK" . "auth") ("-ERR" . "pswd"))
```

Comment:
"capa" is defined as multiple lines.
So, I don't know why your problem happens.

Reply:
My problem happens in "greeting" that is dummy CAPA instead of true CAPA.